newdevtech/newdev_api
# #166172698 Setup Travis CI

#### What does this PR do?
- Sets up Travis CI

#### Description of Task to be completed?
- add a `travis config` file
- add a sample test file
- delete package-lock.json
- add package-lock.json to gitignore
- add test script
- add prettier config to eslint rule

#### How should this be manually tested?
- Run `npm test`
- Confirm that the test passes
- Subsequently, when a PR is raised, Travis CI runs the test files and confirms that they're good to go

#### What are the relevant pivotal tracker stories?
[#166172698](https://www.pivotaltracker.com/story/show/166172698)
